apps/frontend/src/libs/components/table/styles.module.css

```diff
@@ -65,7 +65,7 @@
 	table-layout: fixed;
 }
 
-.table .table-row:last-child .table-data {
+.table-row:last-child .table-data {
```

Comment:
Maybe we need to remove .table for all other selectors here to keep it consistent?

Reply:
Yeah, makes sense. Thank you!